netsim-ffi/netsim.h

```diff
@@ -0,0 +1,97 @@
+#include <cstdarg>
```

Comment:
Usually, C headers need some `#ifdef/#define/#endif` macros to prevent multiple inclusions, should we add those in the `cbindgen` configuration?

Reply:
yes we are missing a cbindgen configuration. @mcsherrylabs can you add one ?